### PR TITLE
Release 10.0.0

### PR DESCRIPTION
**Pull request by jalextowle**
## Description

This PR cuts the `v10` release of 0x-mesh. Changes for this release can be found in the `CHANGELOG.md` file and will also be published on Github. Some highlights that were included in this release are:

- New database implementations for standalone and browser nodes
- Performance improvements to `ordersync` (the initial bootup of Mesh nodes)
- A new GraphQL API

### CODE REVIEW

**Review by z2trillion — APPROVED** [review]
looks good

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,384 +2,398 @@
 
 This changelog is a work in progress and may contain notes for versions which have not actually been released. Check the [Releases](https://github.com/0xProject/0x-mesh/releases) page to see full release notes and more information about the latest released versions.
 
+## v10.0.0
+
+### Breaking changes 🛠
+
+-   Reject orders that have taker addresses that are non-zero and are not whitelisted. [#896](https://github.com/0xProject/0x-mesh/pull/896)
+-   Removed support for Coordinator orders. [#895](https://github.com/0xProject/0x-mesh/pull/895)
+-   The JSON-RPC is no longer supported by Mesh nodes. Only a GraphQL API is supported. [#858](https://github.com/0xProject/0x-mesh/pull/858)
```
Comment (Recmo) [inline CHANGELOG.md:11]: This should be up top!

Comment (Recmo) [reply CHANGELOG.md:11]: Also sounds a bit Dostoyevsky. How about "JSON-RPC supported is removed in favour of the a GraphQL API"/

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,384 +2,398 @@
...
+-   Upgraded to Go 1.15, which contains several WebAssembly performance improvements [#911](https://github.com/0xProject/0x-mesh/pull/911)
+-   Made MAX_BYTES_PER_SECOND configurable for non-bootstrap Mesh nodes. [#879](https://github.com/0xProject/0x-mesh/pull/879)
+-   Added the ability for ordersync requestors to track the progress of individual ordersync connections in case connection is broken temporarily. [#877](https://github.com/0xProject/0x-mesh/pull/877)
+-   Implemented parallelization of ordersync requests. [#848](https://github.com/0xProject/0x-mesh/pull/848)
+-   Optimized the computation of optimal eth_call batch sizes for order validation. [#846](https://github.com/0xProject/0x-mesh/pull/846)
+-   Implemented more aggressive pruning for "removed" orders when database utilization is high [#839](https://github.com/0xProject/0x-mesh/pull/839)
+-   Optimizes conversions from Golang to Javascript. [#820](https://github.com/0xProject/0x-mesh/pull/820)
+-   Implemented new database solutions for both standalone and browser nodes. [#793](https://github.com/0xProject/0x-mesh/pull/793)
```
Comment (Recmo) [inline CHANGELOG.md:23]: Nitpick: This reads more like a commit log than a list of features.

**Review by Recmo — APPROVED** [review]

**Review by jalextowle — COMMENTED** [review]
LGTM

**Pull request merged**